prime-angular-frontend/src/app/modules/enrolment/shared/components/next-steps-infographic/next-steps-infographic.component.html

```diff
@@ -29,10 +34,10 @@ <h5>
     </h5>
     <p>
       <button mat-flat-button
-        type="button"
-        color="primary"
-        *ngIf="hasVCIssuance"
-        (click)="openQR($event)">
+              type="button"
```

Comment:
Just curious why this change occurred? Is there a new linter in play?

Reply:
No someone doesn't have the linter turned on in VSCode.  The HTML entity attributes should always align vertically except on elements that aren't block-level like `<a href="...">...</a>`